Stylesheet/Components/snackbar.css

```diff
@@ -0,0 +1,45 @@
+/* This is scackbar css file */
+
+.snack-container, .appear, #basic-snackbar.show {
```

Comment:
Is this usage of id intentional, or you will refactor it later?

Reply:
I have added some js code to show the snackbar on button click , that's why given the id